src/storage/kura.rs

```diff
@@ -309,68 +309,86 @@ impl Disk {
         Ok(model::Block::from(buffer))
     }
 
-    //TODO: implement reading all blocks
-    fn read_vec(&self) -> Vec<model::Block> {
-        Vec::new()
+    /// returns a sorted vector of blocks starting from 0 height to the top block
+    async fn read_all(&self) -> Vec<model::Block> {
+        let mut height = 0;
+        let mut blocks = Vec::new();
+
+        while let Ok(block) = self.read(height).await {
```

Comment:
Maybe we can read all bytes at once?
https://docs.rs/async-std/1.5.0/async_std/io/trait.Read.html#method.read_to_end

Reply:
`Disk::read` function already internally reads all bytes at once for a given file, or you mean someting different?

Reply:
Sry, forgot that we split blocks into one block one file.